test/transform-stdout.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+/*jshint asi: true */
+
+var test = require('tap').test,
+    exec = require("child_process").exec;
+
+test('\nshould print to stdout with --stdout option', function (t) {
+
+    exec('node doctoc.js test/fixtures/readme-with-custom-title.md --stdout', (error, stdout, stderr) => {
+      if (error) {
+        console.error(`exec error: ${error}`);
+        return;
+      }
+      t.deepEqual(stdout,
+        `
+DocToccing single file "test/fixtures/readme-with-custom-title.md" for github.com.
```

Comment:
Huge fan of template literals, but those won't run with older node versions and I'd like to keep these running.
If you could please pull these out into files in the [fixtures](https://github.com/thlorenz/doctoc/tree/master/test/fixtures) and read them in instead, we'd highly appreciate it :).
I know it's more work, but then again nothing else in this repo uses non ES5 features, so for a bit I'd like to stay backwards compat.